src/nunjucks/moduk/components/header/macro.njk

```diff
@@ -0,0 +1,75 @@
+{#
+ This macro is based on the GOV.UK equivalent from the GOV.UK Frontend:
+ https://github.com/alphagov/govuk-frontend/blob/cdfc281bebf0afb6ea121ee8f52eb332dfc40615/src/govuk/components/header/template.njk
+#}
+
+{% macro modukHeader(params) %}
+
+{% set menuButtonText = params.menuButtonText if params.menuButtonText else 'Menu' %}
```

Comment:
In GOV.UK Frontend the inner part of the macro is put in a separate file. I find that a bit annoying personally as I never know which file to look at.